apps/emqx_utils/src/emqx_utils.erl

```diff
@@ -777,6 +783,23 @@ redact2_test_() ->
     Keys = [secret, passcode],
     [{case_name(atom, Key), fun() -> Case(Key, Checker) end} || Key <- Keys].
 
+redact_is_authorization() ->
```

Comment:
seems the test is not run:

```suggestion
redact_is_authorization_test_() ->
```

but if enabled,it will probably fail on `"Authorization"` case 🤔

Reply:
checking like that would solve this issue:
```
is_sensitive_key(jwt) -> true;
is_sensitive_key("jwt") -> true;
is_sensitive_key(<<"jwt">>) -> true;
is_sensitive_key(authorization) -> true;
is_sensitive_key(Key) when is_binary(Key) orelse is_list(Key) -> is_authorization(Key);
is_sensitive_key(_) -> false;
```

because any `"Authorization"` case would be checked by `is_authorization`, which converts to lowercase to check (the authorization string/atom case could be moved to is_authorization as well).

Reply:
I think the idea is to make a fast check before lowering each binary/string key.